include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -522,13 +523,13 @@ __radix_sort_reorder_submit(_ExecutionPolicy&& __exec, ::std::size_t __segments,
                 }
                 if (__residual > 0)
                 {
-                    _InputT __in_val{};
+                    _ValueT __in_val{};
```

Comment:
I am not sure that `_ValueT` is necessarily default constructible

Reply:
It is was before. I just rename the type name.
Regarding default constructible.
If I am not mistaken that code (residual processing) added by Alexey recently.
Looking into the code I can notice it is radix sort code.. So, we have just an arithmetic type and it is always default constructible.